api-channel-jira/src/main/java/com/synopsys/integration/alert/api/channel/jira/distribution/custom/MessageReplacementValuesBuilder.java

```diff
@@ -0,0 +1,73 @@
+package com.synopsys.integration.alert.api.channel.jira.distribution.custom;
+
+public class MessageReplacementValuesBuilder {
+    private String providerName;
+    private String projectName;
+    private String projectVersionName;
+    private String componentName;
+    private String componentVersionName;
+    private String severity;
+    private String policyCategory;
+
+    public static MessageReplacementValues trivial(String providerLabel, String projectName) {
+        return new MessageReplacementValuesBuilder(providerLabel, projectName)
+                   .projectVersionName(MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE)
+                   .componentName(MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE)
+                   .componentVersionName(MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE)
+                   .severity(MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE)
+                   .policyCategory(MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE)
+                   .build();
+    }
```

Comment:
Since the constructor takes the two required params, I don't think this static initializer is still needed. In the `build()` method, we should have a `defaultIfBlank()` method to handle replacing values with the default as needed.